data-sets.md

```diff
@@ -0,0 +1,6 @@
+# Civic Data Sets
+
+- [Data Gov (US Government's Open Data)] (https://www.data.gov/) 
+- [Kaggle] (https://www.kaggle.com/datasets)
```

Comment:
Would remove Kaggle since these datasets are really meant for a specific topic/challenge. Even if they do have some civic data like US patent proposals and approvals, they are only for a few weeks so that people can train models to predict some behavior